src/RTSolution/CRTM_Active_Sensor.f90

```diff
@@ -508,7 +498,7 @@ SUBROUTINE CRTM_Compute_Reflectivity_TL(Atm, & ! Input
     ELSE
         Height = Atm%Height
     ENDIF
-    dZ_m = (Atm%Height(0:Atm%n_Layers-1) - Atm%Height(1:Atm%n_Layers)) * ONE_THOUSAND
+    dZ_m = (Height(0:Atm%n_Layers-1) - Height(1:Atm%n_Layers)) * ONE_THOUSAND
```

Comment:
You probably want a safety check in here, to ensure that dz_m stays within reasonable bounds.

Reply:
I added a check to make sure that the layer thickness is greater than zero